lib/web/fotorama/fotorama.js

```diff
@@ -1441,6 +1443,13 @@ fotoramaVersion = '4.6.4';
                 return;
             }
 
+            $target = $(e.target);
```

Comment:
Looks like $target was never declared. Also if it used only once - it’s better to inline it’s value

Reply:
$target was declared in the parent function.
Fixed.